label_studio/core/redis.py

```diff
@@ -100,3 +100,16 @@ def start_job_async_or_sync(job, *args, **kwargs):
         return job
     else:
         return job(*args, **kwargs)
+
+
+def is_job_in_queue(queue, func_name, **kwargs):
+    jobs = (
```

Comment:
It should be checked in case job is already in progress on worker

Reply:
@KonstantinKorotaev @makseq  The job doesn't need to be necessarily in progress, right ? It could be in pending status and we still need to avoid new background syncs. My question is: Does the job remain in the queue after executed ?

Reply:
@guilhermemachado26 no, the job will be erased after the execution. Yes, the job can be in queue, but not running. Sounds like we should check all jobs in queue + all running jobs.

Reply:
Sure, adding some tests to cover these cases